cms/static/js/pages/course.js

```diff
@@ -0,0 +1,6 @@
+define(
```

Comment:
I ran into `define cannot be used indirect` problems (https://github.com/webpack/webpack/issues/138) when using this as @cpennington wrote it (wrapped in an IIFE), so I removed the IIFE -- seems to work OK?

Reply:
Yup. Pretty sure I did the same in the container page PR. It wasn't being used by the assets_index page, I believe.